src/containers/LearningResourcePage/components/taxonomy/TopicConnections.jsx

```diff
@@ -176,11 +137,18 @@ class TopicConnections extends Component {
                   openedPaths={openedPaths}
                   structure={structure}
                   toggleOpen={this.handleOpenToggle}
-                  listClass={listStyle}
-                  renderListItems={props =>
-                    this.renderListItems({ ...props, closeModal })
-                  }
-                  fileStructureFilters={fileStructureFilters}
+                  renderListItems={props => (
+                    <StructureButtons
+                      {...props}
+                      closeModal={closeModal}
+                      activeTopics={activeTopics}
+                      availableFilters={availableFilters}
```

Comment:
Ser ikke noe behov for å sende inn activeTopics og availableFilters når du spreader props. Eventuelt ikke spreade props?

Reply:
props er fra renderListItems. Endret navn for å tydeliggjøre